docker-compose.azure.yml

```diff
@@ -0,0 +1,11 @@
+volumes:
+  postgres-data:
+    driver: azure_file
+    driver_opts:
+      share_name: triage-postgres-data
+      storage_account_name: jlltriageportalstorage
```

Comment:
For main branch, should we change this to an env variable?

This could help with the different environments we want to create and the option of testing locally.

Reply:
resolved in changes